packages/lit-dev-content/src/components/litdev-code-language-switch.ts

```diff
@@ -25,45 +24,70 @@ export class LitDevCodeLanguageSwitch extends LitElement {
     :host {
       /* TODO(aomarks) Fix as inline-flex after jsSamples mod is retired. */
       display: var(--litdev-code-language-switch-display, none);
+      font-size: 13px;
+      width: 3.3em;
+      height: 1.6em;
+      border-radius: 1em;
+      padding: 0.1em 0.2em;
+      border: 1.5px solid #ccc;
+      background: white;
+      box-shadow: rgb(0 0 0 / 5%) 0px 0px 1px 1px inset;
       font-family: 'Open Sans', sans-serif;
     }
 
     button {
+      flex: 1;
       display: flex;
+      position: relative;
+      font-size: inherit;
       font-family: inherit;
-      font-weight: inherit;
-      color: black;
-      background: #fbfcff;
-      padding: 2px 14px;
-      border: 1.5px solid #8e9498;
-      transition: background-color 100ms;
-      width: 40px;
-      justify-content: center;
+      background: transparent;
+      border: none;
+      align-items: center;
+      justify-content: space-around;
+      cursor: pointer;
+      padding: 0;
+      z-index: 0;
     }
 
-    /* Note [disabled] implies selected, because the active choice is always
-    disabled.*/
-    button[disabled],
-    button:hover {
-      background: #ebeeff;
-      border-color: #7589ff;
+    #toggle {
+      position: absolute;
+      width: 50%;
+      height: 100%;
+      top: 0;
+      transition: left 100ms;
```

Comment:
Not required but maybe could add [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)? You can sometimes get many moving toggles on the same view.

Reply:
Cool, TIL. Done.